easybuild/easyconfigs/k/Keras/Keras-2.1.3-foss-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'Keras'
+version = '2.1.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://keras.io/'
+description = """Keras is a minimalist, highly modular neural networks library, written in Python and
+capable of running on top of either TensorFlow or Theano."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['7ca3a381523bad40a6922e88951a316664cb088fd01cea07e5ec8ada3327e3c7']
+
+dependencies = [
+    ('Python', '3.6.3'),
+    ('Theano', '1.0.1', versionsuffix),
+    ('h5py', '2.7.1', versionsuffix),
+    ('PyYAML', '3.12', versionsuffix),
+]
+
+# it defaults to Tensorflow
+modextravars = {'KERAS_BACKEND': 'theano'}
```

Comment:
Hmm, this doesn't make much sense anymore since we can now properly install `TensorFlow` from source...

Reply:
fixed in #5821